### PR TITLE
Interpretation help

### PR DESCRIPTION
**Pull request by PromiseDodzi**
This code workflow produces alignments for which we should be glad to understand the results.

### CODE REVIEW

**Comment by LinguList** [comment]
@PromiseDodzi, why do you use the raw data, if my workflow showed how you can use the segmented data with the updated profiles already?

**Comment by LinguList** [comment]
For me, one of the basic problems here is already this crucial step that you produce an orthography profile with a new workflow that is not standard and tested.

**Comment by LinguList** [comment]
So can we instead just align the CLDF data, and then you check the alignments and then we decide if they results are by any means strange?

**Comment by PromiseDodzi** [comment]
> @PromiseDodzi, why do you use the raw data, if my workflow showed how you can use the segmented data with the updated profiles already?

This is not the raw data. This is data that Fabian, Abbie and myself have cleaned manually

**Comment by PromiseDodzi** [comment]
> For me, one of the basic problems here is already this crucial step that you produce an orthography profile with a new workflow that is not standard and tested.

when we run the orthography profile that you created on our cleaned data, there are many unrecognized forms. I thus modified your orthography profile manually to take care of the cleaned data

**Comment by PromiseDodzi** [comment]
> So can we instead just align the CLDF data, and then you check the alignments and then we decide if they results are by any means strange?

I am thinking @IndianaTones will be interested in this suggestion

**Comment by LinguList** [comment]
Okay. So you cleaned the data manually. I showed in the seminar on Zoom, 
how I used the CLEANED data to convert it to CLDF and that this CLDF 
data has the correct segmentations, and I said: please use this CLDF 
data for the future. Now you run another alignment analysis on the data 
that you cleaned, so I wonder why I would go through the pain of 
including this in the CLDF conversion? I thought we agreed that the next 
point of departure is the CLDF? Should I show how to make alignments 
from there, so we can compare them, as I suggested in the email?

So to summarize again:

Raw data -> Cleaning by Abbie and Promise -> CLDF -> Alignments

You should not run any alignments on the data you cleaned, but on the CLDF.

**Comment by LinguList** [comment]
Sorry, there are NO unrecognized forms there. I did not find any. The forms that are unrecognized there are because you keep using a python-internal workflow, not the workflwo I recommended using lexibank and cldfbench. This has been documented in the seminar on Zoom. You should please now really leave the cleaned data as is and stick to the data in the CLDF folder.

**Comment by PromiseDodzi** [comment]
> Okay. So you cleaned the data manually. I showed in the seminar on Zoom, how I used the CLEANED data to convert it to CLDF and that this CLDF data has the correct segmentations, and I said: please use this CLDF data for the future. Now you run another alignment analysis on the data that you cleaned, so I wonder why I would go through the pain of including this in the CLDF conversion? I thought we agreed that the next point of departure is the CLDF? Should I show how to make alignments from there, so we can compare them, as I suggested in the email? So to summarize again: Raw data -> Cleaning by Abbie and Promise -> CLDF -> Alignments You should not run any alignments on the data you cleaned, but on the CLDF.

Alright.
I am sorry I did not understand it this way.
Thanks for the explanation

**Comment by PromiseDodzi** [comment]
> Sorry, there are NO unrecognized forms there. I did not find any. The forms that are unrecognized there are because you keep using a python-internal workflow, not the workflwo I recommended using lexibank and cldfbench. This has been documented in the seminar on Zoom. You should please now really leave the cleaned data as is and stick to the data in the CLDF folder.

Okay.
I will attempt to reproduce the exact same workflow in your seminar presentation, and if there are any issues, I will not hesitate to write to you.
Thank you once again.

**Comment by PromiseDodzi** [comment]
> Please see again, here is the course summary handout.
> 
> [handout-04.pdf](https://github.com/user-attachments/files/15866620/handout-04.pdf)
> 
> To get alignments and the like, you only must follow the steps described in section 3 following.

okay.
Thank you @LinguList

**Comment by LinguList** [comment]
@IndianaTones, why did you merge this PR? It was full of problems. It deletes the CLDF that I created, it adds ITSELF as a submodule (?), it removes the orthography profile that I wrote, etc.

**Comment by IndianaTones** [comment]
> @IndianaTones, why did you merge this PR? It was full of problems. It deletes the CLDF that I created, it adds ITSELF as a submodule (?), it removes the orthography profile that I wrote, etc.

I got a notification to merge. Shall I undo it?

**Comment by LinguList** [comment]
Yes.
